doc/manual/50-feature-details.en.md

```diff
@@ -13,24 +13,22 @@ common patterns.
 
 ### Incremental backup
 
-Barman implements **file-level incremental backup**. Incremental
-backup is a type of full periodic backup which only saves data changes
+Incremental backup is a type of full periodic backup which only saves data changes
 from the latest full backup available in the catalog for a specific
 PostgreSQL server. It must not be confused with differential backup,
 which is implemented by _WAL continuous archiving_.
 
-> **NOTE:** Block level incremental backup will be available in
-> future versions.
-
-> **IMPORTANT:** The `reuse_backup` option can't be used with the
-> `postgres` backup method at this time.
-
 The main goals of incremental backups in Barman are:
 
 - Reduce the time taken for the full backup process
 - Reduce the disk space occupied by several periodic backups (**data
   deduplication**)
 
+Barman currently supports **file-level incremental backups** (using `rsync`) as well
```

Comment:
should we move this higher? @barthisrael @gustabowill

Reply:
Humm I like it here because it leaves an opening to the following sections which discuss these topics in more detail. But Idk, would like to hear more opinions as well.

Reply:
I think it's ok in the current location.